tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Import;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Import\FullyQualifiedStrictTypesFixer
+ */
+final class FullyQualifiedStrictTypesFixerTest extends AbstractFixerTestCase
+{
+    public function testImportedStrictTypesFixWithoutReturn()
+    {
+        $expected = <<<'EOF'
+<?php
+
+use Foo\Bar;
+
+class SomeClass
+{
+    public function doSomething(Bar $foo)
+    {
+    }
+}
+EOF;
+
+        $input = <<<'EOF'
+<?php
+
+use Foo\Bar;
+
+class SomeClass
+{
+    public function doSomething(\Foo\Bar $foo)
+    {
+    }
+}
+EOF;
+
+        $this->doTest($expected, $input);
+    }
+
+    public function testImportedStrictTypesFixWithReturn()
+    {
+        if (PHP_VERSION_ID < 70000) {
+            $this->markTestSkipped('The strict return type : operator is only avaiable from PHP 7.0.');
+        }
```

Comment:
Why not using the dedicated method DocBlock
```php
/**
 * @requires PHP 7.0
 */
```
?

Reply:
Fixed